src/app/parcels/ParcelsPage.tsx

```diff
@@ -115,6 +115,15 @@ const fetchAndFormatParcelTablesData = async (
     return formattedData;
 };
 
+const areDateRangesIdentical = (
+    dateRangeA: DateRangeState,
+    dateRangeB: DateRangeState
+): boolean => {
+    return dateRangeA.from && dateRangeB.from && dateRangeA.to && dateRangeB.to
+        ? dateRangeA.from?.isSame(dateRangeB.from) && dateRangeA.to?.isSame(dateRangeB.to)
+        : false;
```

Comment:
Shouldn't we return true if From is `null` for both, and To is the same for both for example?

Reply:
ok, I've changed the logic accordingly so the following are the 4 possibilities for returning true:

From is the same for both and To is the same for both => true
From is `null` for both and To is the same for both => true
From is the same for both and To is `null` for both => true
From is `null` for both and To is `null` for both => true

any other possible combination will return false